sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample10_MultiLabelClassify.md

```diff
@@ -10,7 +10,7 @@ You can set `endpoint` and `apiKey` based on an environment variable, a configur
 ```C# Snippet:CreateTextAnalyticsClient
 string endpoint = "<endpoint>";
 string apiKey = "<apiKey>";
-var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+TextAnalyticsClient client = new(new Uri(endpoint), new AzureKeyCredential(apiKey));
```

Comment:
wow. Does .NET that fancy now? with only new() with variable type only.

Reply:
Yes, I love this feature, and I think it reads better than used `var`.